service/lambda/runtime/event/apigatewayauthorizerevt/doc.go

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright 2017 Alsanium, SAS. or its affiliates. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+/*
+Package apigatewayproxyevt allows you write AWS Lambda functions as the back end
```

Comment:
Change for correct description. Like:
Package apigatewayproxyevt allows you to write an Amazon API Gateway custom authorizer that provide control access to your API methods.